src/com/intellij/idea/plugin/hybris/runConfigurations/LocalSapCXRunSettingsEditor.kt

```diff
@@ -0,0 +1,147 @@
+/*
+ * This file is part of "SAP Commerce Developers Toolset" plugin for IntelliJ IDEA.
+ * Copyright (C) 2019-2024 EPAM Systems <hybrisideaplugin@epam.com> and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.intellij.idea.plugin.hybris.runConfigurations
+
+import com.intellij.compiler.options.CompileStepBeforeRun
+import com.intellij.diagnostic.logging.LogsGroupFragment
+import com.intellij.execution.ExecutionBundle
+import com.intellij.execution.JavaRunConfigurationExtensionManager
+import com.intellij.execution.ui.BeforeRunFragment
+import com.intellij.execution.ui.CommonParameterFragments
+import com.intellij.execution.ui.RunConfigurationFragmentedEditor
+import com.intellij.execution.ui.SettingsEditorFragment
+import com.intellij.idea.plugin.hybris.common.HybrisConstants
+import com.intellij.idea.plugin.hybris.common.utils.HybrisI18NBundleUtils
+import com.intellij.openapi.externalSystem.service.execution.configuration.addBeforeRunFragment
+import com.intellij.openapi.externalSystem.service.execution.configuration.addEnvironmentFragment
+import com.intellij.openapi.externalSystem.service.execution.configuration.fragments.SettingsEditorFragmentContainer
+import com.intellij.openapi.externalSystem.service.execution.configuration.fragments.addLabeledSettingsEditorFragment
+import com.intellij.openapi.externalSystem.service.ui.util.LabeledSettingsFragmentInfo
+import com.intellij.openapi.observable.properties.AtomicProperty
+import com.intellij.openapi.observable.util.bind
+import com.intellij.ui.components.fields.ExtendableTextField
+import com.intellij.ui.components.fields.IntegerField
+import org.jetbrains.idea.maven.execution.MavenRunConfiguration
```

Comment:
`maven` is an optional plugin dependency, do we need it here?

Reply:
No, as you have written below, it is a copy past issue. Will fix this